addons/account/static/src/components/many2many_tags_banks/many2many_tags_banks.js

```diff
@@ -6,6 +6,22 @@ import { registry } from "@web/core/registry";
 import { TagsList } from "@web/core/tags_list/tags_list";
 import { _t } from "@web/core/l10n/translation";
 import { onMounted } from "@odoo/owl";
+import { Many2XAutocomplete } from "@web/views/fields/relational_utils";
+
+export class Many2ManyTagsBanksAutocomplete extends Many2XAutocomplete {
+    async loadOptionsSource(request) {
+        // Always include Search More
+        let options = await super.loadOptionsSource(...arguments);
```

Comment:
i see that it's a copy paste on what we did in Many2XTaxTagsAutocomplete, could you check with habib that made this component why we do the slice and not just check the number of record ? seems overkill ahah